README.MD

```diff
@@ -1,5 +1,12 @@
  # SCCameraView
 
+  SCCameraView is a camera preview library, which uses google's Camera1 and Camera2 API respective to the device's operating system version. It also allows you take a picture, record videos with both cameras and to toogle between aspect ratios and preview sizes supported by the given sensor. 
```

Comment:
SCCameraView is a camera preview library, which uses Android' Camera1 and Camera2 API respective to the device's operating system version. It allows you take a picture, record videos with both cameras and to toggle between aspect ratios and preview sizes supported by the given sensor.